app/mailers/views/hypertension_fast_track_pilot_mailer.erb.html

```diff
@@ -5,12 +5,12 @@
   </head>
   <body>
     <p>
+      A single-issue <%= @disability_code %> claim for increase was submitted on va.gov.<br/>
       <% if @rrd_claim_processed %>
-        A single-issue <%= @disability_code %> claim for increase was submitted on va.gov.<br/>
         A health summary PDF was generated and added to the claim's documentation.
       <% else %>
-        A single-issue <%= @disability_code %> claim for increase was submitted on va.gov.<br/>
-        There was not sufficient data to generate a health summary PDF associated with this claim. 
+        There was some issue (such as insufficient health data or pending EP) that prevents
+        a health summary PDF to be generated for this claim.
```

Comment:
Is there no way to infer the exact cause here? I'm guessing not, just wondering if it's something we'd be able to determine later in the future

Reply:
We could run the same `pending_ep?` code but that's inefficient. Yang is working on storing the reason with the submission record so we can use for this emailer, as well as diagnostics.